.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+<!-->Please title your PR according to eslint commit conventions</!-->
+<!-->See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details</!-->
+<!-->e.g. "Fix: Semi rule incorrectly flagging extra semicolon (fixes #840)"</!-->
+<!-->or "Upgrade: Esprima to 1.2, switch to using comment attachment (fixes #730)"</!-->
+
+<!-->Link the PR to the original issue</!-->
+<!-->e.g. fixes #123</!-->
+
+<!-->Delete Fix, New, Breaking sections as appropriate</!-->
+### Fix
+* A sentence describing each fix
+
+### New
+* A sentence describing each new feature
+
+### Breaking
+* A sentence describing each breaking change
+
+<!-->List appropriate steps for testing if needed</!-->
+### Testing
+1. Steps for testing
+
+<!-->Mention any other dependencies</!-->
+<!-->e.g. requires #456</!-->
```

Comment:
Comment syntax didn't work for me, have switched to a different one.
Expanded title examples.
Added link examples.
Changed Tag example to match our processes (Fix, Update, New, Breaking).

```suggestion
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)
[//]: # (Title examples:)
[//]: # (Revision 0.0.x "Fix: Corrected rendering bug with titles \(fixes #123\)")
[//]: # (Revision 0.0.x "Update: Switched to ES6 \(fixes #123\)")
[//]: # (Minor 0.x.0 "New: Added tooltip API \(fixes #123\)")
[//]: # (Major x.0.0 "Breaking: Rewritten API \(fixes #123\)")

[//]: # (Link the PR to the original issue)
[//]: # (fixes #123)
[//]: # (refs #123)

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### Fix
* A sentence describing each fix

### Update
* A sentence describing each udpate

### New
* A sentence describing each new feature

### Breaking
* A sentence describing each breaking change

[//]: # (List appropriate steps for testing if needed)
### Testing
1. Steps for testing

[//]: # (Mention any other dependencies)
[//]: # (requires #456)

```